source/Ship.cpp

```diff
@@ -630,7 +618,15 @@ void Ship::FinishLoading(bool isNewInstance)
 				count -= eit->second;
```

Comment:
This section up here can be restated as well

Reply:
The code here looks correct to me as-is (but it now just re-uses the `equipped` variable from line 570 instead of the class variable), what do you feel should be changed here?